cmd/osbuild-worker/jobimpl-osbuild-koji.go

```diff
@@ -169,7 +170,6 @@ func (impl *OSBuildKojiJobImpl) Run(job worker.Job) error {
 	// copy pipeline info to the result
 	result.PipelineNames = args.PipelineNames
 
-	err = job.Update(&result)
 	if err != nil {
 		return fmt.Errorf("Error reporting job result: %v", err)
 	}
```

Comment:
Shouldn't the error handling also be removed?

Reply:
of course, thanks!